dropwizard-logging/src/main/java/smartthings/dw/logging/KVLogger.java

```diff
@@ -8,69 +8,69 @@
 public class KVLogger {
 	private KVLogger() {}
 
-	public static void debug(Logger l, String key, Map<String, Object> params) {
+	public static void debug(Logger l, String key, Map<String, ? extends Object> params) {
```

Comment:
What's the problem we're seeing?  everything extends object so this looks like it would be unnecessary?